examples/amp-story/interactive_slider.html

```diff
@@ -0,0 +1,200 @@
+      }
+      h1 {
+        color: black;
+        font-size: 30px;
+        margin-bottom: 10px;
+        text-align: center;
+        font-family: 'Poppins';
+      }
+      p {
+        font-family: 'Poppins';
+        font-size: 15px;
+      }
+      .bottom {
+        position: absolute;
+        top: 80%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        width: 22em;
+      }
+    </style>
+    <style amp-boilerplate>
+      body {
+        -webkit-animation: -amp-start 8s steps(1, end) 0s 1 normal both;
+        -moz-animation: -amp-start 8s steps(1, end) 0s 1 normal both;
+        -ms-animation: -amp-start 8s steps(1, end) 0s 1 normal both;
+        animation: -amp-start 8s steps(1, end) 0s 1 normal both;
+      }
+      @-webkit-keyframes -amp-start {
+        from {
+          visibility: hidden;
+        }
+        to {
+          visibility: visible;
+        }
+      }
+      @-moz-keyframes -amp-start {
+        from {
+          visibility: hidden;
+        }
+        to {
+          visibility: visible;
+        }
+      }
+      @-ms-keyframes -amp-start {
+        from {
+          visibility: hidden;
+        }
+        to {
+          visibility: visible;
+        }
+      }
+      @-o-keyframes -amp-start {
+        from {
+          visibility: hidden;
+        }
+        to {
+          visibility: visible;
+        }
+      }
+      @keyframes -amp-start {
+        from {
+          visibility: hidden;
+        }
+        to {
+          visibility: visible;
+        }
+      }
+    </style>
+    <noscript
+      ><style amp-boilerplate>
+        body {
+          -webkit-animation: none;
+          -moz-animation: none;
+          -ms-animation: none;
+          animation: none;
+        }
+      </style></noscript
+    >
+    <link rel="canonical" href="." />
+    <title>Slider</title>
+  </head>
+  <body>
+    <amp-story
+      standalone
+      publisher="The Publisher"
+      title="My Story"
+      publisher-logo-src="logo.png"
+      poster-portrait-src="poster-portrait.png"
+      poster-square-src="poster-square.png"
+      poster-landscape-src="poster-landscape.png"
+    >
+      <amp-story-page id="cover">
+        <amp-story-grid-layer template="fill">
+          <amp-img
+            layout="fixed"
+            width="1600"
+            height="1600"
+            src="https://images.unsplash.com/photo-1587270613304-4cc9ef012b92?ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTl8fHN1cGVyaGVybyUyMGNvbWljJTIwYm9va3xlbnwwfHwwfHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=700&q=60"
+          />
+        </amp-story-grid-layer>
+        <amp-story-grid-layer template="vertical">
+          <div class="title">
+            <h1>Slider Component</h1>
+            <p>This is the most basic form of slider in the light theme, with a custom accent color</p>
+          </div>
+
+          <amp-story-interactive-slider
+            style="--interactive-accent-color: #651ffe"
+            class="bottom"
```

Comment:
Please remove the `bottom` class from all components. 
It's important to see how the component looks with it's default styles within a grid layer.
We can center the component in a follow up PR.